screens/ImageScreen.js

```diff
@@ -0,0 +1,206 @@
+import React from 'react';
+import {
+  ActivityIndicator,
+  Button,
+  Clipboard,
+  Image,
+  StatusBar,
+  StyleSheet,
+  Text,
+  TouchableOpacity,
+  View,
+} from 'react-native';
+import * as ImagePicker from 'expo-image-picker';
+import * as firebase from 'firebase';
+import * as Permissions from 'expo-permissions';
+import KitText from '../components/KitText';
+import KitButton  from '../components/KitButton'
+import Colors from '../constants/Colors';
+import FontStyles from '../constants/FontStyles';
+import uuid from 'uuid';
+
+export default class ImageScreen extends React.Component{
+  state = {
+    image: null,
+    //TODO: SUBMIT BUTTON CHANGE COLOR BASED ON IF PHOTO WAS SELECTED
+  };
+  
+  async componentDidMount() {
+    await Permissions.askAsync(Permissions.CAMERA_ROLL);
```

Comment:
I'm not sure if `componentDidMount()` is the right place to put this, as it will as for permission whenever this component changes (unless `askAsync()` only asks once and never again but having an async on each reload might not be best performance wise). 

@kristielim 

I'm going to go with the "if it works it works" mentality, but let's add a `TODO` comment so we can revisit this in the future